main.go

```diff
@@ -197,11 +197,7 @@ type _escFile struct {
 }
 
 func (_escLocalFS) Open(name string) (http.File, error) {
-	f, present := _escData[path.Clean(name)]
-	if !present {
-		return nil, os.ErrNotExist
-	}
-	return os.Open(f.local)
```

Comment:
This method could work,
If file exists in static, request the local path, else request the file by appending there prefix.